src/tests/ftest/server/metadata.py

```diff
@@ -164,18 +163,18 @@ def test_metadata_addremove(self):
         Use Cases:
             ?
 
-        :avocado: tags=metadata,metadata_free_space,nvme,small
+        :avocado: tags=metadata,metadata_free_space,nvme,small,hw,full_regression
         """
         self.pool.pool.connect(2)
         for k in range(10):
             container_array = []
-            self.d_log.debug("Container Create Iteration {}".format(k))
+            self.log.info("Container Create Iteration {} / 10".format(k))
             for cont in range(NO_OF_MAX_CONTAINER):
                 container = DaosContainer(self.context)
                 container.create(self.pool.pool.handle)
                 container_array.append(container)
 
-            self.d_log.debug("Container Remove Iteration {} ".format(k))
+            self.log.info("Container Remove Iteration {} / 10".format(k))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments